examples/llm/mcloud/mcli-1b-eval.yaml

```diff
@@ -0,0 +1,82 @@
+
```

Comment:
Maybe accidentally deleted the top of the file?

Reply:
integrations are at the top of the other mcloud examples that I'm seeing. I think the issue is just the extra new lines?

Reply:
ok, no problem. I think we may have landed on different ordering conventions for mcloud yamls in different places. but not an issue to solve here. just removing the new lines is fine here.